demo/admin/src/pages/PageContentBlock.tsx

```diff
@@ -27,6 +27,7 @@ export const PageContentBlock = createBlocksBlock({
         textImage: TextImageBlock,
         damVideo: DamVideoBlock,
         youTubeVideo: YouTubeVideoBlock,
+        vimeoVideo: VimeoVideoBlock,
```

Comment:
Should we just remove the individual video blocks from the page content and just use the OneOfBlock? Would be more closely to an actual project

Reply:
Would this breaks some project?

Reply:
I guess not because its demo 🤔

Reply:
https://github.com/vivid-planet/comet/pull/2068/commits/d7ec3f216637dc7a646cb263b5a0432375d24751